lib/server/services/qualityControlFlag/QcFlagService.js

```diff
@@ -120,6 +122,35 @@ class QcFlagService {
         return dplDetector;
     }
 
+    /**
+     * Update given effective periods of some QC flags
```

Comment:
Which ones?! Randomly? Hard coded? `Remove a time segment from a list of QC flags effective periods`

Reply:
applied